### PR TITLE
Add max limit config option

### PR DESCRIPTION
**Pull request by jk-tripathy**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -7,6 +7,9 @@ CONTAINER_IMAGE=/netscratch/enroot/nvcr.io_nvidia_pytorch_23.07-py3.sqsh
 # chooses a Slurm partition
 PARTITION=RTXA6000-SLT
 
+# set the max time limit for the job
```
Comment (ArneBinder) [inline .env.example:10]: describe the format of the time limit. What does the example value of `3-0` mean?

**Review by ArneBinder — CHANGES_REQUESTED** [review]
One issue, please see below.

File: .env.example (hunk 1 above)
Comment (jk-tripathy) [reply .env.example:10]: got it, ill add a short description on how the time format is set in slurm

Comment (ArneBinder) [reply .env.example:10]: thanks!

**Review by ArneBinder — APPROVED** [review]

**Comment by ArneBinder** [comment]
lgtm

**Pull request merged**